artcommon/artcommonlib/runtime.py

```diff
@@ -54,6 +54,8 @@ def initialize_logging(self):
         self._logger = logging.getLogger('artcommonlib')
 
     def initialize_konxflux_db(self):
+        if self.initialized or self.konflux_db:
```

Comment:
I am not seeing where `self.initialized` is being set to True. It works anyway because you're checking for `self.konflux_db` to be != None, but maybe it makes sense to set `self.initialized = True` at the end of the initialization function

Reply:
I was following initialize_logging (https://github.com/openshift-eng/art-tools/blob/eed0c8e6d883a2c2cc6473f48f60f3a03dd86691/artcommon/artcommonlib/runtime.py#L38). However we expect self.initialized to be set by the child class's initialize function.

I don't plan to make a lot of changes for the Runtime. It seems make more sense to me to remove the check for self.initialized.